From 9fcd477efaa975f1b76f9ff5be1b950488afb3cd Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 13 Oct 2016 14:40:13 -0700
Subject: [PATCH] IMGAPI-591 updates.jo can't update manifests of images
 migrated in CM-712 Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
 Approved by: Todd Whiteman <todd.whiteman@joyent.com>

---
 bin/imgapi-standalone-restore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/bin/imgapi-standalone-restore b/bin/imgapi-standalone-restore
index a2bfe29..ba960e4 100755
--- a/bin/imgapi-standalone-restore
+++ b/bin/imgapi-standalone-restore
@@ -156,6 +156,7 @@ elif [[ $infoStatus != "200" ]]; then
 else
     echo "Restore /data/imgapi/images"
     $MANTASYNC -r $bakDir/images /data/imgapi/images
+    chown -R nobody:nobody /data/imgapi/images
 fi
 
 echo ""
@@ -168,6 +169,7 @@ elif [[ $infoStatus != "200" ]]; then
 else
     echo "Restore /data/imgapi/manifests"
     $MANTASYNC -r $bakDir/manifests /data/imgapi/manifests
+    chown -R nobody:nobody /data/imgapi/manifests
 fi
 
 # The restored.marker file is used to tell 'imgapi-standalone-backup' that it
-- 
2.21.0

